### PR TITLE
First hcmi-es-model 

### PR DESCRIPTION
**Pull request by rosibaj**
I cleaned up some copy-past issues.  Please let me know any feedback!

### CODE REVIEW

File: es-model-archive/hcmi-es-model-latest/model_centric.mapping.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        "date_created": {
+          "type": "date",
+          "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss"
+        },
+        "date_updated": {
+          "type": "date",
+          "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss"
+        },
+        "chemotherapeutic_drugs": {
+          "type": "boolean"
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/model_centric.mapping.json]: The field name `chemotherapeutic_drugs` seems suggest it's a list of drug names used for chemotherapy. If that's the case, the type should be `keyword`

File: es-model-archive/hcmi-es-model-latest/model_centric.mapping.json [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                },
+                "histopathological_biomarkers": {
+                  "type": "keyword"
+                }
+              }
+        },
+        "disease_status": {
+          "type": "keyword"
+        },
+        "file": {
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/model_centric.mapping.json]: `file` to `files`

File: es-model-archive/hcmi-es-model-latest/model_centric.mapping.json [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+                 } 
+              }
+        },
+        "gender": {
+          "type": "keyword"
+        },
+        "growth_rate": {
+          "type": "long"
+        },
+        "licensing_requirements": {
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/model_centric.mapping.json]: `licensing_requirements` does not sound like a boolean. If it is boolean, would `licensing_required` or `is_licensing_required` be a better name?

File: es-model-archive/hcmi-es-model-latest/model_centric.mapping.json [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+          "type": "keyword"
+        },
+        "source_model_url": {
+          "type": "keyword"
+        },
+        "source_sequence_url": {
+          "type": "keyword"
+        },
+        "split_ratio": {
+          "type": "keyword"
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/model_centric.mapping.json]: that sounds like a numeric value

File: es-model-archive/hcmi-es-model-latest/model_centric.mapping.json [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+        },
+        "ssm": {
+          "type": "nested",
+          "properties": {
+              "gene": {
+                "type": "keyword"
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/model_centric.mapping.json]: `gene_symbol` should be better here

File: es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+  "mappings": {
+    "model_centric": {
+      "properties": {
+        "id": {
+          "type": "keyword"
+        },
+        "gene": {
+          "type": "keyword"
+        },
+        "model": {
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json]: `models`?

File: es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json [hunk 7]
```diff
@@ -0,0 +1,156 @@
+{
+  "mappings": {
+    "model_centric": {
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json]: `model_centric` => `ssm_centric`

File: es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+      "properties": {
+        "id": {
+          "type": "keyword"
+        },
+        "gene": {
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json]: `gene` => `gene_symbol`

File: es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+                "date_created": {
+                  "type": "date",
+                  "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss"
+                },
+                "date_updated": {
+                  "type": "date",
+                  "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss"
+                },
+                "chemotherapeutic_drugs": {
+                  "type": "boolean"
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json]: same comment as in `model_centric` mapping

File: es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+                  "type": "keyword"
+                },
+                "gender": {
+                  "type": "keyword"
+                },
+                "growth_rate": {
+                  "type": "long"
+                },
+                "licensing_requirements": {
+                  "type": "boolean"
```
Comment (junjun-zhang) [inline es-model-archive/hcmi-es-model-latest/ssm_centric.mapping.json]: same comment as in model_centric mapping

File: es-model-archive/hcmi-es-model-latest/model_centric.mapping.json (hunk 1 above)
Comment (rosibaj) [reply es-model-archive/hcmi-es-model-latest/model_centric.mapping.json]: The value in this field is yes/no.  Yes indicates that there is a list of chemotherapeutic drugs available if you purchase the model. but the list of chemotherapeutic drugs will not be available yet.

Comment (junjun-zhang) [reply es-model-archive/hcmi-es-model-latest/model_centric.mapping.json]: would they be available later? if so, you may want to reserve the name `chemotherapeutic_drugs` for the actual drugs, and use `treated_with_chemotherapy` (or something close) as the name of this boolean field.

File: es-model-archive/hcmi-es-model-latest/model_centric.mapping.json (hunk 4 above)
Comment (junjun-zhang) [reply es-model-archive/hcmi-es-model-latest/model_centric.mapping.json]: ok, keep it `keyword`

**Pull request merged**